### PR TITLE
docs: Update V1 neopixel module support for RGBW

### PR DESCRIPTION
**Pull request by microbit-carlos**
Based on the changes added to the V2 documentation from PR https://github.com/bbcmicrobit/micropython/pull/694

Remove any mention to V2. Kept the rest of the text as close as possible to reduce conflicts when rebasing the `v2-docs` branch.

Review URL: https://microbit-micropython--736.org.readthedocs.build/en/736/neopixel.html

### CODE REVIEW

**Comment by microbit-carlos** [comment]
@microbit-mark could you review this one as well once you get a chance? Thanks!

**Comment by microbit-carlos** [comment]
Review URL: https://microbit-micropython--736.org.readthedocs.build/en/736/neopixel.html

File: docs/neopixel.rst [hunk 1]
```diff
@@ -50,12 +52,18 @@ Classes
...
 
 .. py:class::
-    NeoPixel(pin, n)
+    NeoPixel(pin, n, bpp=3)
 
     Initialise a new strip of ``n`` number of neopixel LEDs controlled via pin
-    ``pin``. Each pixel is addressed by a position (starting from 0). Neopixels
-    are given RGB (red, green, blue) values between 0-255 as a tuple. For
-    example, ``(255,255,255)`` is white.
+    ``pin``. To support RGBW neopixels, so a third argument can be passed to
```
Comment (microbit-mark) [inline docs/neopixel.rst:58]: I think removing `so` makes this sentence make more sense.

```suggestion
    ``pin``. To support RGBW neopixels, a third argument can be passed to
```

**Comment by microbit-mark** [comment]
@microbit-carlos one suggested change, otherwise LGTM 👍🏻

**Comment by microbit-carlos** [comment]
Awesome, thanks Mark!

**Pull request merged**